DQM/HcalTasks/plugins/QIE11Task.cc

```diff
@@ -134,6 +140,32 @@ QIE11Task::QIE11Task(edm::ParameterSet const& ps):
 	}
 }
 
+
+/* virtual */ bool QIE11Task::_isApplicable(edm::Event const& e)
+{
+  if (_ptype!=fOnline || _laserType < 0)
+    return true;
+  else
```

Comment:
if you don't use {} for the first block, don't use it for the else-block either. It looks rather confusing....